webplugin/js/app/km-message-markup-1.0.js

```
@@ -52,146 +50,144 @@ Kommunicate.messageTemplate = {
-                    data.fileNameToShow = data.fileMeta.name.replace("AWS-ENCRYPTED-", "");
+                    data.fileNameToShow = data.fileMeta.name.replace('AWS-ENCRYPTED-', '');
                     data.blobKey = data.fileMeta.blobKey;
                 }
                 return Mustache.to_html(
-                    Kommunicate.messageTemplate.getAttachmentApplicationTemplate(
-                        data
-                    ),
+                    Kommunicate.messageTemplate.getAttachmentApplicationTemplate(data),
                     data
                 );
                 break;
```

Biome (1.9.4)

[error] This code is unreachable

... because this statement will return from the function beforehand


(lint/correctness/noUnreachable)